### PR TITLE
[REVIEW] Fix SeriesGroupBy.nunique() to return a Series

### PR DESCRIPTION
**Pull request by shwina**
Sorry for the multiple closed PRs prior to this one, had trouble retargeting to branch 0.15 from branch 0.16.

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# cuDF 0.16.0 (Date TBD)
+
+## New Features
+
+## Improvements
+
+## Bug Fixes
+
+- PR #5921 Fix SeriesGroupBy.nunique() to return a Series
```
Comment (kkraus14) [inline CHANGELOG.md:9]: Changelog needs to move to branch-0.15

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# cuDF 0.16.0 (Date TBD)
+
+## New Features
+
+## Improvements
+
+## Bug Fixes
+
+
```
Comment (kkraus14) [inline CHANGELOG.md:9]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -266,6 +266,7 @@
 - PR #5319 Disallow SUM and specialize MEAN of timestamp types
 - PR #5797 Fix a missing data issue in some Parquet files
 - PR #5787 Fix column create from dictionary column view
+- PR #5921 Fix SeriesGroupBy.nunique() to return a Series
```
Comment (kkraus14) [inline CHANGELOG.md:269]:
```suggestion
- PR #5926 Fix SeriesGroupBy.nunique() to return a Series
```

Comment (shwina) [reply CHANGELOG.md]: Thanks, @kkraus14 :)

**Comment by kkraus14** [comment]
rerun tests

**Comment by kkraus14** [comment]
rerun tests

**Comment by kkraus14** [comment]
rerun tests

**Pull request merged**